l10n_it_fattura_accompagnatoria/__manifest__.py

```diff
@@ -1,14 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 Simone Vanin - Agile Business Group
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

Comment:
Credo che la licenza consigliata sia `AGPL`, soprattutto visto https://github.com/OCA/l10n-italy/issues/1383.
Hai qualche motivo particolare per usare `LGPL` o potresti modificare in `AGPL`?

Reply:
Era la versione che c'era sul repository in openerp-italia, modifico in `AGPL`